### DISCUSSION TITLE
Im upgrading my Prisma version from 4.10 to 5.2, but when i run "npx prisma generate", i have an issue: "Downloading Prisma engines for Node-API for windows [==========          ] 49%Error: EPERM: operation not permitted". Can someone help me ?

### DISCUSSION THREAD

**Discussion by Helios15** [prisma/prisma#21041] [Q&A]
### Bug description

Downloading Prisma engines for Node-API for windows [==========          ] 49%Error: EPERM: operation not permitted, unlink 'npm\node_modules\prisma\node_modules\@prisma\engines\schema-engine-windows.exe'

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by janpio** [comment]
You still seem to have a process running that is blocking the Query Engine from being deleted. Windows does not allow replacing the file while it is being used.

**Comment by janpio** [comment]
Already asked in https://github.com/prisma/prisma/discussions/21031
